src/main/java/com/google/devtools/build/lib/bazel/SpawnLogModule.java

```diff
@@ -162,7 +162,11 @@ public void afterCommand() throws AbruptExitException {
         spawnLogContext.close();
         if (!outputStreams.isEmpty()) {
           InputStream in = rawOutput.getInputStream();
-          StableSort.stableSort(in, outputStreams);
+          if (spawnLogContext.needSort()) {
+            StableSort.stableSort(in, outputStreams);
+          } else {
+            StableSort.ignoreSort(in, outputStreams);
```

Comment:
It's a little odd to have a method on `StableSort` that doesn't sort at all. Can you just inline `ignoreSort` here?

Reply:
To clarify, what I meant was to move the `if (shouldSort) { stableSort(...) } else { ... }` logic into the `SpawnLogModule` class, and leave only the `stableSort` method in the `StableSort` class.